dataCollection/settings.py.template

```diff
@@ -207,39 +201,11 @@ LOGGING = {
         },
         'django.request': {
             'handlers': ['request_handler'],
-            #'level': 'ERROR',
              'level': 'DEBUG',
-            #'propagate': True
              'propagate': False
-        },
-        #'django': {
-        #    'handlers': ['logfile'],
-        #    'level': 'ERROR',
-        #    'propagate': False,
-        #}
+        }
     }
 }
-#LOGGING = {
-#    'version': 1,
-#    'disable_existing_loggers': False,
-#    'filters': {
-#        'require_debug_false': {
-#            '()': 'django.utils.log.RequireDebugFalse'
-#        }
-#    },
-#    'handlers': {
-#        'mail_admins': {
-#            'level': 'ERROR',
-#            'filters': ['require_debug_false'],
-#            'class': 'django.utils.log.AdminEmailHandler'
-#        },
-#        'console': {
-#            'level': 'DEBUG',
-#            'class': 'logging.StreamHandler',
-#        },
-#    }
-#}
-
 
 KEY = '52af783df367b7a5c7d01457d82544bf'
 SALT = '58268845878249858254353589767595'
```

Comment:
Turns out I can't comment on it because it wasn't changed, but we have an API key at the bottom of this file as well.  While we're removing keys, we should take a look at that